### PR TITLE
Add references to other NLD-useful packages

### PR DESCRIPTION
**Pull request by Datseris**

### CODE REVIEW

File: docs/src/contents.md [hunk 1]
```diff
@@ -101,3 +96,11 @@ And, in order to automatically deduce dimensions, we also offer methods for:
     * Fine-tuning of the algorithms that compute the above (e.g. Theiler window and many more)
     * [Windowed RQA](@ref) of the above
 * [Recurrence Networks](@ref)
+
+
+## Other NLD-relevant packages
+Besides DynamicalSystems.jl, the Julia programming language has a thriving ecosystem with plenty of functionality that is relevant for nonlinear dynamics. We list some useful references below:
+
+* [DifferentialEquations.jl](https://diffeq.sciml.ai/dev/index.html) - Besides providing solvers for standard ODE systems (same infastructure used for DynamicalSystens.jl), it also has much more things like SDE solvers or uncertainty quantification.
+* [DiffEqSensitivity.jl](https://github.com/SciML/DiffEqSensitivity.jl) - Sensitivity analysis, i.e., derivatives of the solutions of ODEs, or functions of the solutions, versus parameters.
```
Comment (frankschae) [inline docs/src/contents.md:105]:
```suggestion
* [DiffEqSensitivity.jl](https://github.com/SciML/DiffEqSensitivity.jl) - Discrete and continuous local sensitivity analysis, i.e., derivatives of the solutions of ODEs, or functions of the solutions, versus parameters, hosting [various forward and adjoint methods as well as methods tailored to chaotic systems](https://diffeq.sciml.ai/stable/analysis/sensitivity/).
* [GlobalSensitivity.jl](https://github.com/SciML/GlobalSensitivity.jl) Global sensitivity analysis assessing the effect of any input variables over a larger domain on the output.
```

Comment (frankschae) [reply docs/src/contents.md:105]: I probably would have written something along the lines suggested above for the sensitivity packages. Perhaps  

- (Bayesian) [parameter inference packages](https://diffeq.sciml.ai/stable/analysis/parameter_estimation/) or 
- [probabilistic numerical ODE solvers](https://github.com/nathanaelbosch/ProbNumDiffEq.jl)

could be mentioned as well. Maybe @ChrisRackauckas has some additional ideas.

Comment (frankschae) [reply docs/src/contents.md:105]: I probably would have written something along the lines suggested above for the sensitivity packages. Perhaps  

- (Bayesian) [parameter inference packages](https://diffeq.sciml.ai/stable/analysis/parameter_estimation/) or 
- [probabilistic numerical ODE solvers](https://github.com/nathanaelbosch/ProbNumDiffEq.jl)

could be mentioned as well. Maybe @ChrisRackauckas has some additional ideas.

File: docs/src/contents.md [hunk 2]
```diff
@@ -101,3 +96,12 @@ And, in order to automatically deduce dimensions, we also offer methods for:
...
 * [Recurrence Networks](@ref)
+
+
+## Other NLD-relevant packages
+Besides DynamicalSystems.jl, the Julia programming language has a thriving ecosystem with plenty of functionality that is relevant for nonlinear dynamics. We list some useful references below:
+
+* [DifferentialEquations.jl](https://diffeq.sciml.ai/dev/index.html) - Besides providing solvers for standard ODE systems (same infastructure used for DynamicalSystens.jl), it also has much more things like SDE solvers or uncertainty quantification.
+* [DiffEqSensitivity.jl](https://github.com/SciML/DiffEqSensitivity.jl) - Discrete and continuous local sensitivity analysis, i.e., derivatives of the solutions of ODEs, or functions of the solutions, versus parameters, hosting [various forward and adjoint methods as well as methods tailored to chaotic systems](https://diffeq.sciml.ai/stable/analysis/sensitivity/).
+* [GlobalSensitivity.jl](https://github.com/SciML/GlobalSensitivity.jl) Global sensitivity analysis assessing the effect of any input variables over a larger domain on the output.
+* [BifurcationKit.jl](https://github.com/rveltz/BifurcationKit.jl) - Featureful toolkit for _automated_ bifurcation analysis.
```
Comment (Datseris) [inline docs/src/contents.md:107]:
```suggestion
* [BifurcationKit.jl](https://github.com/rveltz/BifurcationKit.jl) - Featureful toolkit for _automated_ bifurcation analysis.
* [NetworkDynamics.jl](https://github.com/PIK-ICoNe/NetworkDynamics.jl) - Package for easily simulating dynamics on networks and transforming network systems into `ODEProblem` (that can be made directly into a `ContinuousDynamicalSystem`).
```

**Pull request merged**